examples/flex-flows/unify-ai/.env.example

```diff
@@ -0,0 +1,2 @@
+UNIFY_AI_API_KEY=<your_Unify_AI_api_key>
```

Comment:
@crazygao I guess we should suggest to put this in integrations folder?

Reply:
Yes, please.

Reply:
put this folder under examples/flows/integrations[https://github.com/microsoft/promptflow/tree/main/examples/flows/integrations]

Reply:
moved.